uint16 item_id
---
string name
